### PR TITLE
extend test_compress_command to cover B3's base.html

### PR DESCRIPTION
**Pull request by orangejenny**
https://github.com/dimagi/commcare-hq/pull/9742 wasn't caught by tests because
a) the script tag was inside a `compress js` instead of a `block js`
b) the templates generated by DjangoParser.parse don't have their names set

@benrudolph

### CODE REVIEW

File: corehq/apps/style/tests/test_compress_command.py [hunk 1]
```diff
@@ -65,11 +74,15 @@ def test_compress_offline(self):
             if self._is_b3(filename):
                 with open(filename, 'r+') as f:
                     for line in f.readlines():
-                        if (BLOCK_JS in line or BLOCK_CSS in line) and ENDBLOCK not in line:
+                        has_start_tag = BLOCK_JS in line or BLOCK_CSS in line
```
Comment (dimagimon) [inline corehq/apps/style/tests/test_compress_command.py]: W291 trailing whitespace

**Comment by czue** [comment]
nice! and it caught a bug? :+1: when tests pass

**Comment by benrudolph** [comment]
thanks :+1: after tests

**Comment by orangejenny** [comment]
fixed some more tests, good to go now

**Pull request merged**